test/router_ics_skf_worker_SUITE.erl

```diff
@@ -90,287 +75,159 @@ end_per_testcase(TestCase, Config) ->
-                    {devaddrs, [<<X, 0, 0, 72>>]},
-                    {keys, [{crypto:strong_rand_bytes(16), crypto:strong_rand_bytes(16)}]}
-                ],
-                router_device:new(ID)
-            )
-        end,
-        lists:seq(1, 2)
+    RemoveAll = router_ics_skf_worker:pre_remove_all(),
+    ok = router_ics_skf_worker:remove_all(
+        RemoveAll,
+        fun(Msg) -> ct:print("remove all progress: ~p", [Msg]) end
     ),
 
-    %% Add a device without any devaddr or session, it should be filtered out
-    meck:expect(router_device_cache, get, fun() ->
-        lager:notice("router_device_cache:get()"),
-        BadDevice = router_device:new(<<"bad_device">>),
-        [BadDevice | Devices0]
-    end),
+    ?assertEqual({ok, []}, router_ics_skf_worker:remote_skf()),
 
-    %% Add a fake device to config service, as it is not in Router's cache it should get removed
-    router_test_ics_skf_service:send_list(
-        #iot_config_session_key_filter_v1_pb{
-            oui = 0,
-            devaddr = 0,
-            session_key = <<>>
-        },
-        true
-    ),
+    ok.
 
-    %% Check after reconcile, we should get the list and then 3 updates, (1 remove and 2 adds)
-    [{Type3, Req3}, {Type2, Req2}, {Type1, Req1}, {Type0, _Req0}] = rcv_loop([]),
-    [Device1, Device2] = Devices0,
-
-    ?assertEqual(list, Type0),
-    ?assertEqual(update, Type1),
-    ?assertEqual(remove, Req1#iot_config_session_key_filter_update_req_v1_pb.action),
-    ?assertEqual(
-        #iot_config_session_key_filter_v1_pb{
-            oui = 0,
-            devaddr = 0,
-            %% Can be a binary, but empty string fields are lists
-            session_key = []
-        },
-        Req1#iot_config_session_key_filter_update_req_v1_pb.filter
-    ),
-    ?assertEqual(update, Type2),
-    ?assertEqual(add, Req2#iot_config_session_key_filter_update_req_v1_pb.action),
-    ?assertEqual(
-        #iot_config_session_key_filter_v1_pb{
-            oui = router_utils:get_oui(),
-            %% Config service talks of devaddrs of BE
-            devaddr = binary:decode_unsigned(binary:decode_hex(<<"48000001">>)),
-            session_key = erlang:binary_to_list(binary:encode_hex(router_device:nwk_s_key(Device1)))
-        },
-        Req2#iot_config_session_key_filter_update_req_v1_pb.filter
-    ),
-    ?assertEqual(update, Type3),
-    ?assertEqual(add, Req3#iot_config_session_key_filter_update_req_v1_pb.action),
-    ?assertEqual(
-        #iot_config_session_key_filter_v1_pb{
-            oui = router_utils:get_oui(),
-            %% Config service talks of devaddrs of BE
-            devaddr = binary:decode_unsigned(binary:decode_hex(<<"48000002">>)),
-            session_key = erlang:binary_to_list(binary:encode_hex(router_device:nwk_s_key(Device2)))
-        },
-        Req3#iot_config_session_key_filter_update_req_v1_pb.filter
+reconcile_skf_test(_Config) ->
+    %% ok = application:set_env(router, test_udpate_skf_delay_ms, 500),
```

Comment:
do these comments need to be removed or uncommented? if the latter, the same typo could trip someone up later on